events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
-  - [Communication Channels and Property Access](#communication-channels-and-property-access)
+Comms Leads are responsible for leading the comms team, training shadows, and ensuring all comms activities are planned, prioritized, and completed successfully. They are committed to run the comms team for the current cycle and advise as Emeritus KCS Comms Lead in the following comms cycle (2 cycles, 1 year). Must have [Kubernetes Org Membership]. Expected to attend in-person.
+
+Shadows will learn both how to do comms and how to run the KCS Comms team, as directed by the Comms Lead. They are committing to be involved with the event team for 1.5 years or 3 comms cycles (shadow, lead, and emeritus). Must either have [Kubernetes Org Membership] or be planning to pursue org membership.
+
+Volunteers are expected to assist with comms during the current KCS cycle, as directed by the lead. Volunteers do not need to have or be pursuing [Kubernetes Org Membership].
+
+An emeritus comms lead is a past lead who provides mentorship to the current comms lead. Should have [Kubernetes Org Membership].
+
+The Website Manager is responsible for ensuring updates to the [Kubernetes Contributor Website](k8s.dev) happen. Requires [Kubernetes Org Membership]
+
+- [KCS Comms Lead Overview](#kcs-comms-lead-overview)
+- [Shadow to the KCS Comms Lead Role](#shadow-to-the-kcs-comms-lead-role)
+- [KCS Comms Volunteer Overview](#kcs-comms-volunteer-overview)
+- [KCS Website Manager Overview](#kcs-website-manager-overview)
+- [Emeritus KCS Comms Lead Overview](#kcs-website-manager-overview)
+- [Comms Rules and Guidelines](#comms-rules-and-guidelines)
 - [Documentation](#documentation)
 
-## Overview
+## KCS Comms Lead Overview
+
+A KCS Comms Lead should plan to be involved with KCS planning for at least a year (2 KCS events), first as lead (for 1 or more events), and then as an emeritus mentor to the succeeding Comms lead (for 1 or more events). The workload of being an Emeritus Comms Lead is light, but they are still actively involved.
+
+A KCS Comms Lead holds that role from the beginning of planning for a specific KCS event until all activities for that event are completed, including post-event activities such as event retros and updating the handbook (this doc). This timeframe is the "cycle" for that KCS event.
 
 As Communications Lead, you are responsible for the overall communications
 before, during, and after the event. The role also includes ensuring all non-technical content (web, mail,
 physical prints) is correct and informative, and supporting the other roles
 (registration, content) as needed.
 
+A Comms Lead should plan to attend their KCS event in-person. Onsite, a Comms lead acts as the contact person for any comms needed onsite, such as messaging about room or registration changes that may occur onsite. In the event that a KCS Comms Lead is unable to attend in-person, the lead is responsible for designating an in-person comms point of contact.
```

Comment:
```suggestion
A Comms Lead should plan to attend their KCS event in-person. Onsite, a Comms lead acts as the contact person for any comms needed onsite, such as messaging about room or registration changes that may occur onsite. In the event that a KCS Comms Lead is unable to attend in-person, the lead is responsible for designating an in-person comms point of contact, that have access to the various comms channels. 
```

The person on site need to act fast so they should have access or rights to publish